crates/burrego/src/opa/builtins/regex.rs

```diff
@@ -117,9 +117,9 @@ impl Display for ExpressionList {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         let mut result = String::new();
         for expression in self.0.iter() {
-            result.push_str(&format!("{}", expression));
+            write!(f, "{}", expression)?;
         }
-        write!(f, "{}", result)
+        Ok()
```

Comment:
```suggestion
        Ok(())
```
Does it compile just with Ok()? I think it should be Ok(()). We can't see it in the CI as it failed because policy-sdk 0.6.1 was not available at that time

Reply:
Indeed you are right, whoops. Amended.